### ISSUE TITLE
pt.compute does not raise error when names appear more then once

### ISSUE THREAD

**Issue by pindakaas42** [Amber-MD/pytraj#1553]
I am often using pt.compute and i noticed that this: 
```
import pytraj as pt
traj = pt.datafiles.load_tz2()
data = pt.compute(['rms myrms refindex 0 @CA','radgyr myrms @CA nomax','molsurf',], traj, ref=traj[3])
```
will not raise an error. But instead print an error message to the screen. 

Catching the error on the library site is quite bad because this way pytraj silently fails, without me being alarmed about it.

**Comment by hainm** [comment]
I will consider it. Under the hood, above commands are `cpptraj''s commands and `pytraj` just send them directly to `cpptraj` and retrieve the output from `cpptraj`. But you might be right. Let me see if it's easier send the error from C++ to pytraj in this case to raise error earlier.

**Comment by hainm** [comment]
fixed in #1554 

thanks @pindakaas42
